src/hardware/voodoo.cpp

```diff
@@ -929,53 +930,54 @@ struct draw_state
 
 struct triangle_worker
 {
-	bool threads_active, use_threads, disable_bilinear_filter;
+	std::atomic_bool threads_active;
+	bool use_threads, disable_bilinear_filter;
 	UINT16 *drawbuf;
 	poly_vertex v1, v2, v3;
 	INT32 v1y, v3y, totalpix;
 	SDL_sem* sembegin[TRIANGLE_THREADS];
-	volatile bool done[TRIANGLE_THREADS];
+	std::atomic_bool done[TRIANGLE_THREADS];
 };
 
 struct voodoo_state
 {
-	UINT8				chipmask;				/* mask for which chips are available */
+	UINT8				chipmask = {};				/* mask for which chips are available */
```

Comment:
Since we retouched these lines, please do replacements like `UINT8` -> `uint8_t` (int retouched lines only, please!). Once finished run `./scripts/format-commit.sh` to apply our coding standards to retouched lines (I think it will nicely align all the comments).